backend/app/main.py

```
@@ -72,8 +79,26 @@ async def seed_users():
+        except Exception as exc:  # noqa: BLE001 — Bridge 실패가 BE 기동 막지 않음
+            logging.getLogger(__name__).warning(
+                "ai_agent_bridge.start_failed err=%s", exc
+            )
+
     yield
+
+    if bridge is not None:
+        try:
+            await bridge.stop()
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)